ddtrace/appsec/_handlers.py

```diff
@@ -20,6 +20,19 @@
 _BODY_METHODS = {"POST", "PUT", "DELETE", "PATCH"}
 
 
+def _get_content_length(environ):
+    content_length = environ.get("CONTENT_LENGTH")
+    transfer_encoding = environ.get("HTTP_TRANSFER_ENCODING")
+
+    if transfer_encoding == "chunked" or content_length is None:
+        return None
+
+    try:
```

Comment:
```suggestion
    if not content_length:
        return 0
    try:
```


Technically it could be `None`, which `int(None)` would give us a `TypeError`.

We could do this approach, `int(content_length or '')` or add `TypeError` to the except statement.

I think this is a bit more clear on intent and avoids an unnecessary exception being raised in this case.